packs/metaweather/actions/get_location_results.py

```diff
@@ -0,0 +1,18 @@
+import requests
+from st2actions.runners.pythonrunner import Action
+
+
+__all__ = [
+    'GetLocationResultsAction'
+]
+
+
+class GetLocationResultsAction(Action):
+    def run(self, query):
+        results = requests.get('https://www.metaweather.com/api/location/search/?query=%s' % (query))
+
+        if results.status_code != 200:
+            raise Exception("Call to MetaWeather failed.")
```

Comment:
Should also include body (or at least log it) in this case otherwise it will be very hard to debug it.